packages/components/src/popover/utils.js

```diff
@@ -181,18 +181,47 @@ export function computePopoverYAxisPosition(
 	stickyBoundaryElement,
 	anchorRef,
 	relativeOffsetTop,
-	forcePosition
+	forcePosition,
+	editorWrapper
 ) {
 	const { height } = contentSize;
 
 	if ( stickyBoundaryElement ) {
 		const stickyRect = stickyBoundaryElement.getBoundingClientRect();
-		const stickyPosition = stickyRect.top + height - relativeOffsetTop;
-
-		if ( anchorRect.top <= stickyPosition ) {
+		const stickyPositionTop = stickyRect.top + height - relativeOffsetTop;
+		const stickyPositionBottom =
+			stickyRect.bottom - height - relativeOffsetTop;
```

Comment:
Nitpick: we can move this inside the `if ( ! isRoomAboveInCanvas )`, as it's not used anywhere else.
Not a big deal either way, so feel free to disregard this comment!

Reply:
The `stickyPositionTop` is used outside that condition, since it is the sticky position for the fallback/standard behavior.  `stickyPositionBottom` isn't used outside that condition, and we could move it in there.. but originally I thought it might make sense to define these two variations next to each other as it feels organized.  Happy to move it down if you still think that makes more sense.

Reply:
Yup I was talking of `bottom` only, but keeping it outside for organization purposes works for me!